inloop/solutions/validators.py

```diff
@@ -0,0 +1,13 @@
+from constance import config
+
+
+def validate(file_name):
+    file_name = file_name.lower()
+    for ending in parse_allowed_file_endings():
+        if file_name.endswith(ending):
+            return True
+    return False
+
+
+def parse_allowed_file_endings():
```

Comment:
For users outside of this module, `parse_` is confusing. It should be `get_allowed_file_endings()`.